internal/services/loadbalancer/backend_address_pool_address_resource_test.go

```diff
@@ -58,7 +58,7 @@ func TestAccBackendAddressPoolAddressDisappears(t *testing.T) {
 	})
 }
 
-func TestAccBackendAddressPoolAddressUpdate(t *testing.T) {
+func TestAccRegionalBackendAddressPoolAddressUpdate(t *testing.T) {
```

Comment:
we should not be putting the "catagory" in front, it should ALWAYS be the resource name
```suggestion
func TestAccBackendAddressPoolAddress_regional_update(t *testing.T) {
```

as well use a _ to seperate them. please align all tests to be this way